src/DGSM_Vi.jl

```diff
@@ -0,0 +1,50 @@
+
+mutable struct DGSM_Vi
+    a::Float64
+    absa::Float64
+    asq::Float64
+end
+
+"""
+The inputs to the function DGSM_Vi are as follows:
+1.f: 
+    This is the input function based on which the values of DGSM are to be evaluated
+    Eg- f(x) = x[1]+x[2]^2
+        This is function in 2 variables
+2.samples:
+    Depicts the number of sampling set of points to be used for evaluation of E(a), E(|a|) and E(a^2)
+    a = partial derivative of f wrt x_i
+3.distri:
+    Array of distribution of respective variables
+    Eg- dist = [Normal(5,6),Uniform(2,3)]
+	for two variables
+"""
+function DGSM_Vi(f,samples,distr)
```

Comment:
create a dispatch that makes it easy to use with a `DEProblem`. See Sobol and Morris interfaces for details there.

Reply:
Actually, do this in a follow up PR.